controller/tap/server.go

```diff
@@ -482,16 +482,25 @@ func indexPodByIP(obj interface{}) ([]string, error) {
 // Since errors encountered while hydrating metadata are non-fatal and result
 // only in missing labels, any errors are logged at the WARN level.
 func (s *server) hydrateEventLabels(ev *public.TapEvent) {
-	err := s.hydrateIPLabels(ev.Source.Ip, ev.SourceMeta.Labels)
+	labels := ev.GetSourceMeta().GetLabels()
+	if labels == nil {
+		labels = make(map[string]string)
```

Comment:
Does this work?  I would expect that this creates an empty map but doesn't set it into the source meta struct.

Reply:
oh _that's_ what this is doing.  Yes, good catch. I'll try again.